tutorials/distributed-calculator/go/Dockerfile

```
@@ -1,8 +1,7 @@
 #first stage - builder
-FROM golang:1.15-buster as builder
+FROM golang:1.19-buster as builder
```

```suggestion
FROM golang:1.22-bullseye as builder
```
What if we go straight to supported version 1.22 on bullseye so long as there's no breaking changes?  Feels like being on latest supported is what we want if we're changing this.

---

Yes, I fully agree. I was playing safe with 1.19.

---

Ok since it's testing ok, let's go with the 1.22, bullseye and bookworm changes.  Are you ok with updating your PR for this?

---

Used the same `bookworm` version for both build and run containers.